### PR TITLE
Add Contributing.md

### LINKED ISSUE #20: GitHub Workflow and Quality Policy Docs
Define a standardized workflow, how to contribute, PR formatting/commentation, commit formatting/commentaion, etc.

### PR DESCRIPTION
**Pull request by bananabrann**
Closes #20 

- Adds everything we talked about in #20 
- Needs specs on Quality of Code from @KevinThePepper or @stucamp 

For formatted version, check out `pierson-dev` branch

What do y'all think?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+#### Dev Branch
+The dev branch contains the *latest source code* for the Flameboi Slack API. It is a protected branch, and is the default branch you will submit pull requests to when submitting your contributions.
+
+#### Master Branch
+The master branch is the *production code* that Flameboi is currently opperating on. Whatever is in master is what will be on Slack. Pull requests from dev to master will **only be intiated and managed by the project manager(s)**.
+
+## Contribution Requirements
+Observe the following in your git flow:
+- Make an Issue prior to working on your code. We don't want you to work on something that is actually banned or already implemented!
+- Your personal development branches must start with your name and be consistent. E.g., `bobby-dev`, `bobby-new-feature-here`
```
Comment (stucamp) [inline CONTRIBUTING.md:35]: isn't the -dev a trigger to have github enforce safety?  not sure

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+#### Checkout Existing Branch
+```
+git checkout existing-branch
+```
+
+#### Checkout a New Branch
+```
+git checkout -B your-new-branch
+```
+
```
Comment (stucamp) [inline CONTRIBUTING.md:73]: Maybe add section about git add <file changed> and git commit -m "did this here to fix this" and doing these things for each bug/fix you're doing... if the fix impact multiple files i'm probably okay with grouping those, but specifically would want to see incremental commits pertaining to specific issues.  I realize that I, myself have a bad habit of doing that so it'll be a good goal for me too!

**Review by stucamp — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (stucamp) [reply CONTRIBUTING.md:35]: nix that... just tested... -dev is all gravy!!!

**Review by stucamp — APPROVED** [review]
with the caveat that once we nail down commit standards and scope of commits we'll add them here at that time!

**Review by KevinThePepper — APPROVED** [review]
We'll talk more about standardizing the code quality. This is probably a club-wide standard we'll want to establish

**Pull request merged**